test/unit/app/utils/test_utils_user.py

```diff
@@ -140,3 +148,29 @@ def test_send_welcome_sms(mocker, test_client, init_database, mock_sms_apis,
     messages = mock_sms_apis
     assert messages == [{'phone': '+61123456789', 'message': expected_welcome},
                         {'phone': '+61123456789', 'message': expected_terms},]
+
+def test_attach_transfer_account_to_user(test_client, init_database, create_organisation):
+    with db.session.no_autoflush:
```

Comment:
why is this needed?

Reply:
I am interested in what you would make of these errors where the autoflush.

Reply:
sorry I meant the autoflush - and we discucsed already :)